tests/server/blueprints/cases/test_matchmaker_controllers.py

```diff
@@ -131,9 +134,6 @@ def test_matchmaker_add_sv(app, user_obj, case_obj, sv_variant_obj, mocker):
     # GIVEN an app containing MatchMaker connection params
     with app.test_client() as client:
         # GIVEN a SV variant with genes
-        sv_variant_obj["samples"] = [
```

Comment:
Fixed the fixture instead. The only reason I had issues with this was that I tried to skip the var_type fallback first, since we usually get variant categories from the file loading type anyway. This test happened to be one of the ones that barked. I gave up on it later, just too many tests as it is, and kept the previous fallback. 😊